### ISSUE TITLE
"themes" filter must have a matching alias defined

### ISSUE THREAD

**Issue by mandahylla** [tattersoftware/codeigniter4-themes#26]
Hello got this issue when i'm trying to setup and test the module.
it comes when i add themes filter in the route : $routes->get('/', 'Login::index', ['filter' => 'themes']);
i've respect any step in the readme.
![Capture tatter themes](https://github.com/tattersoftware/codeigniter4-themes/assets/321008/177ecdfc-2194-46ac-9c16-88fe83dbff74)

**Comment by MGatner** [comment]
You are totally right: the README is missing the step to add an alias. I will update that, but in the meantime you can get that info from the framework docs: https://codeigniter4.github.io/CodeIgniter4/incoming/filters.html#aliases

**Comment by MGatner** [comment]
Actually this is supposed to be handled by the Filters discovery: https://github.com/tattersoftware/codeigniter4-themes/blob/develop/src/Config/Filters.php#L11

Do you have autoloading of modules on?
